library/oneview_logical_interconnect_group.py

```diff
@@ -139,10 +139,10 @@
     type: dict
 '''
 
-from ansible.module_utils.oneview import OneViewModuleBase, OneViewModuleResourceNotFound
+from ansible.module_utils.oneview import OneViewModule, OneViewModuleResourceNotFound
 
 
-class LogicalInterconnectGroupModule(OneViewModuleBase):
+class LogicalInterconnectGroupModule(OneViewModule):
     MSG_CREATED = 'Logical Interconnect Group created successfully.'
     MSG_UPDATED = 'Logical Interconnect Group updated successfully.'
     MSG_DELETED = 'Logical Interconnect Group deleted successfully.'
```

Comment:
These constants are not used in this file.

Reply:
These class variables are used by resource_present and resource_absent methods in the base class. Base class having these variables, here we are overwriting these variables for resource-specific messages.